data/templates/default.latex

```diff
@@ -156,6 +156,20 @@ $if(CJKmainfont)$
   \fi
 $endif$
 \fi
+%% Support for special Unicode characters
+% Zero width non-joiner
+\def\zerowidthnnonjoiner{%
```

Comment:
What's the reason of using an additional 'n' character in the name?

Reply:
Late night coding :D